src/test/regress/split_schedule

```diff
@@ -1,6 +1,6 @@
 # Split Shard tests.
 # Include tests from 'minimal_schedule' for setup.
-test: multi_test_helpers multi_test_helpers_superuser columnar_test_helpers
+test: multi_test_helpers multi_test_helpers_superuser
```

Comment:
Note to reviewer:

Removed columnar_test_helpers from citus schedules so that we don't have to create citus_columnar in those schedules